docs/source/contributing/releasing.adoc

```diff
@@ -28,8 +21,38 @@ defined in your environment as _GITHUB_ACCESS_TOKEN_.
   For example, issues with a merged pull requests should be labeled as *resolution/done*.
 . Close milestone.
 
+[[auto-relase]]
+== Automated Release
+
+An automated release can be performed as:
+
+----
+$ export API_KEY=<api-key>
+$ export RELEASE_VERSION=<expected release version> # Eg : 1.0.0 (without 'v')
```

Comment:
I think we should determine the milestone id, from the release version. Using the GitHub API this should be possible. Right now, I can accidentally release a version against the wrong milestone. Since we have a one to one relationship we should be able to determine this.